extension/entitystore/ec2Info.go

```diff
@@ -99,40 +103,6 @@ func (ei *EC2Info) setInstanceIDAccountID() error {
 	}
 }
 
-func (ei *EC2Info) setAutoScalingGroup() error {
```

Comment:
why are we removing autoscaling group all together?

Reply:
This function is basically a retry loop which calls `retrieveAsgName` infinitely until success. We already have helper struct for this type of operation which is why I used [limitedRetryer](https://github.com/aws/amazon-cloudwatch-agent/pull/1440/files#diff-d1bdee0ab819bf459c37069c1cd13cc77105541449f620965e0bfd60c7e100fbR55). Though I just realized original code keeps retrying until successful. This one retries until successful then continues to refresh on an interval. Will flip `oneTime` option to true for the retryer.